### PR TITLE
Add rki_covid integration docs

### PR DESCRIPTION
**Pull request by thebino**
## Proposed change

Add initial rki_covid integration docs

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/66431
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/rki_covid.markdown [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: rki_covid
+    districts:
+     - name: 'SK Augsburg'
+     - name: 'SK Berlin Mitte'
+     - name: 'BL Bayern'
+     - name: 'LK München'
+```
```
Comment (frenck) [inline source/_integrations/rki_covid.markdown]: We don't allow YAML configuration for new integration that connects to devices or services.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/rki_covid.markdown (hunk 1 above)
Comment (thebino) [reply source/_integrations/rki_covid.markdown]: Thank you, removed.

File: source/_integrations/rki_covid.markdown [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  - sensor
+---
+
+This integration allows to visualize the covid numbers provided by the [Robert Koch-Institut](https://www.rki.de). Multiple sensors clustered by districts express detailed information on local regions and areas.
+
+
+
+|Sensor  |Type|Description
+|:-----------|:---|:------------
+|`sensor.NAME_count`| number | indicates the confirmed cases.
```
Comment (frenck) [inline source/_integrations/rki_covid.markdown]: Entity IDs are not guaranteed, in general it is better to describe what sensors are provided instead.

File: source/_integrations/rki_covid.markdown [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+Open the `Configuration` of your Home Assistant instance and select `Integrations`.
+Add a new integration, search and select `rki covid`.
+A dialog appears to select a district or state to monitor.
+After submitting the dialog, the newly added sensor entities are available.
+
+**Hint:** Repeat the process to add multiple districts or states.
+
+![Integration used within the lovelace](/images/integrations/rki_covid/lovelace_graph.png)
+
+{% include integrations/config_flow.md %}
```
Comment (frenck) [inline source/_integrations/rki_covid.markdown]:
```suggestion
{% include integrations/config_flow.md %}

![Integration used within the lovelace](/images/integrations/rki_covid/lovelace_graph.png)
```

File: source/_integrations/rki_covid.markdown [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+## Configuration
+Open the `Configuration` of your Home Assistant instance and select `Integrations`.
+Add a new integration, search and select `rki covid`.
+A dialog appears to select a district or state to monitor.
+After submitting the dialog, the newly added sensor entities are available.
+
+**Hint:** Repeat the process to add multiple districts or states.
+
+![Integration used within the lovelace](/images/integrations/rki_covid/lovelace_graph.png)
```
Comment (frenck) [inline source/_integrations/rki_covid.markdown]: This is not a standard configuration; the result of this is that the user might think something is wrong. E.g., they add the integration and don't get this out of the box...

File: source/_integrations/rki_covid.markdown [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+ha_release: '2021.12'
+ha_config_flow: true
+ha_codeowners:
+  - '@thebino'
+ha_domain: rki_covid
+ha_platforms:
+  - sensor
+---
+
+This integration allows to visualize the covid numbers provided by the [Robert Koch-Institut](https://www.rki.de). Multiple sensors clustered by districts express detailed information on local regions and areas.
```
Comment (frenck) [inline source/_integrations/rki_covid.markdown]: The integration provides data, it does not visualize it...

File: source/_integrations/rki_covid.markdown (hunk 4 above)
Comment (thebino) [reply source/_integrations/rki_covid.markdown]: So the lovelace example should be removed or just a note added to tell users, what a card could look like?

File: source/_integrations/rki_covid.markdown [hunk 6]
```diff
@@ -13,43 +13,37 @@ ha_platforms:
...
-|`sensor.NAME_hospitalizationIncidenceSenior` | float | Hospitalization incidence for people between 60-79 |
-|`sensor.NAME_hospitalizationCasesOld` | float | Hospitalization cases for people older than 80 |
-|`sensor.NAME_hospitalizationIncidenceOld` | float | Hospitalization incidence for people older than 80 |
-
-## Configuration
-Open the `Configuration` of your Home Assistant instance and select `Integrations`.
-Add a new integration, search and select `rki covid`.
-A dialog appears to select a district or state to monitor.
-After submitting the dialog, the newly added sensor entities are available.
+|`count`| number | indicates the confirmed cases.
```
Comment (frenck) [inline source/_integrations/rki_covid.markdown]: These are still code fenced, which indicates it is code? Is that intended?

Comment (thebino) [reply source/_integrations/rki_covid.markdown]: I don't understand? The count is the number of confirmed cases in the sensors district.

Comment (frenck) [reply source/_integrations/rki_covid.markdown]: Sure, but is is code fenced with backticks, so it is code? What does it mean?

Comment (thebino) [reply source/_integrations/rki_covid.markdown]: Ah, you mean the single quotes. I can remove them.

Comment (frenck) [reply source/_integrations/rki_covid.markdown]: Those are backticks, not single quotes 😉 

Anyways, the point being: "hospitalizationIncidenceMerged" <- That looks like code. Can we make it a human form?

Comment (thebino) [reply source/_integrations/rki_covid.markdown]: Do you have any recommendations? I think sensors do not support sub-sensor like structure?

Comment (frenck) [reply source/_integrations/rki_covid.markdown]: Just provide titles and descriptions of the sensors provided?

Comment (thebino) [reply source/_integrations/rki_covid.markdown]: like
| title | description |
|:-----|:--------|
| merged hospitalization incidence | Calculated hospitalization incidence of all states merged together |

Comment (frenck) [reply source/_integrations/rki_covid.markdown]: Yes, but as a list, not table; as tables are hard to consume on mobile devices.

Comment (thebino) [reply source/_integrations/rki_covid.markdown]: okay, now as a list.

File: source/_integrations/rki_covid.markdown (hunk 4 above)
Comment (frenck) [reply source/_integrations/rki_covid.markdown]: I would suggest removing it. Keeping things like this up to date is a challenge.

File: source/_integrations/rki_covid.markdown [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+- **hospitalizationCasesGrown**: Hospitalization cases for people between 35-59 
+- **hospitalizationIncidenceGrown**: Hospitalization incidence for people between 35-59 
+- **hospitalizationCasesSenior**: Hospitalization cases for people between 60-79 
+- **hospitalizationIncidenceSenior**: Hospitalization incidence for people between 60-79 
+- **hospitalizationCasesOld**: Hospitalization cases for people older than 80 
+- **hospitalizationIncidenceOld**: Hospitalization incidence for people older than 80 
+
+**Hint:** Repeat the process to add multiple districts or states.
+
+{% include integrations/config_flow.md %}
```
Comment (frenck) [inline source/_integrations/rki_covid.markdown]:
```suggestion

{% include integrations/config_flow.md %}

**Hint:** Repeat the process to add multiple districts or states.

## Provided sensors

- Confirmed cases
- New confirmed cases
- The numbers of confirmed deaths
- The numbers of new confirmed deaths
- The numbers of confirmed recoveries
- The numbers of new confirmed recoveries
- Confirmed cases per 100.000 inhabitants
- The week incidence per 100.000 inhabitants
- Hospitalization cases for people in Germany summarized  
- Hospitalization incidence for people in Germany summarized 
- Hospitalization cases for people between 00-04 
- Hospitalization incidence for people between 00-04 
- Hospitalization cases for people between 05-14 
- Hospitalization incidence for people between 05-14 
- Hospitalization cases for people between 15-34 
- Hospitalization incidence for people between 15-34 
- Hospitalization cases for people between 35-59 
- Hospitalization incidence for people between 35-59 
- Hospitalization cases for people between 60-79 
- Hospitalization incidence for people between 60-79 
- Hospitalization cases for people older than 80 
- Hospitalization incidence for people older than 80 
```

File: source/_integrations/rki_covid.markdown (hunk 4 above)
Comment (thebino) [reply source/_integrations/rki_covid.markdown]: done

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Comment by thebino** [comment]
@frenck The parent has been changed to https://github.com/home-assistant/core/pull/66431
So this documentation PR is still necessary. Could it be re-opened?

**Comment by frenck** [comment]
@thebino Sure it can, but the parent needs to be updated too.. Please link the parent up

**Comment by thebino** [comment]
@frenck rebase completed ✅

**Pull request closed without merging**